content/push/index.textile

```diff
@@ -78,4 +80,38 @@ Each push target device is associated with a unique @deviceId@ and authenticatio
 
 * By employing a standard "Ably key":/auth/basic or "Token":/auth/token, with a @deviceIdentityToken@, a credential generated during registration to assert the device's identity, included in the request header.
 
-The sevice credential management is handled by the "Ably SDK":https://ably.com/docs/sdks, removing the need for the client application to manage device credentials unless accessing the "push admin API":/api/realtime-sdk/push-admin directly via HTTP.
\ No newline at end of file
+The sevice credential management is handled by the "Ably SDK":https://ably.com/docs/sdks, removing the need for the client application to manage device credentials unless accessing the "push admin API":/api/realtime-sdk/push-admin directly via HTTP.
+
+h4(#Error). Error handling
+
+The @[meta]log:push@ channel logs push notification "metadata":/metadata-stats/metadata. This dedicated channel allows users to monitor system events and errors, providing deeper insights into the system's operations. It helps track the system's state, debug issues, and ensure that important events are logged and can be reviewed.
+
+Metachannels, such as @[meta]log:push@, publish events and errors that aren't otherwise available to clients. It's important to note that client-returned errors will not be published to this channel.
+
+The following example subscribes to the @[meta]log:push@ channel:
+
+```[javascript]
+const channel = ably.channels.get('[meta]log:push');
+```
+
+An example scenario for receiving error logs is when a push notification cannot be delivered.
+
+The following example shows the error logs from @[meta]log:push@ for a failed push notification delivery:
+
+```[json]
+{
+  ErrorInfo: Connection failed
```

Comment:
This example is all for a connection failure, not a failure to deliver a notification.

Reply:
There isn't an error in our FAQ that falls under "failure to deliver a notification".

Reply:
The text above the example says "failed push notification delivery" though. I'd assume a connection failure would be logged to `[meta]log` rather than `[meta]log:push` though, too?